pages/api/login/index.ts

```diff
@@ -28,7 +28,6 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       error: 'Incorrect email or password',
     });
   }
-
-  res.setHeader('Set-Cookie', getTokenCookie(user._id.toString(), req.headers.host))
-    .status(200).json({ success: true });
+  const cookie = getTokenCookie(user._id.toString(), req.headers.host);
```

Comment:
Sorry unsure why this was checked in, all I did here was separate out into a variable so it shouldn't affect anything.